ember/app/templates/map/moderations/index.hbs

```diff
@@ -7,6 +7,32 @@
     {{/link-to}}
   </div>
 
+  <div class="panel-headers">
```

Comment:
The Flag section should be below the moderations. Since the moderations are the main title of the tabs and URL, it makes sense to have that as the first thing you see when viewing the panel.

Reply:
👌